TWTValidation/JSON Validator/Parser/TWTJSONRemoteSchemaManager.m

```diff
@@ -0,0 +1,95 @@
+//
+//  TWTJSONRemoteSchemaManager.m
+//  TWTValidation
+//
+//  Created by Jill Cohen on 10/19/15.
+//  Copyright © 2015 Ticketmaster Entertainment, Inc. All rights reserved.
+//
+
+#import "TWTJSONRemoteSchemaManager.h"
+
+#import <TWTValidation/TWTJSONSchemaASTCommon.h>
+#import <TWTValidation/TWTJSONSchemaParser.h>
+
+
+@interface TWTJSONRemoteSchemaManager ()
+
+@property (nonatomic, strong) NSMutableDictionary *filePathsToJSONSchemaTopLevelNodes;
+
+@end
+
+
+@implementation TWTJSONRemoteSchemaManager
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        _filePathsToJSONSchemaTopLevelNodes = [[NSMutableDictionary alloc] init];
+    }
+    return self;
+}
+
+
+- (BOOL)attemptToConfigureFilePath:(NSString *)filePath onReferenceNode:(TWTJSONSchemaReferenceASTNode *)referenceNode
```

Comment:
I see what you mean about setting the file path. What about:
`- (BOOL)loadSchemaForReferencePath:(NSString *)referencePath filePath:(NSString *)outFilePath pathComponents:(NSString *)outPathComponents error:(NSError *)outError`

Then the parser can manage actually setting the properties, but know it's a valid file that produced a schema/node.

I do like adding an error, so the description can differentiate between 1) a file not found 2) a file that doesn't contain a valid schema